tslint.json

```diff
@@ -59,7 +59,11 @@
     "no-string-throw": true,
     "encoding": true,
     "no-floating-promises": true,
-    "no-implicit-dependencies": true,
+    "no-implicit-dependencies": [
+      true,
+      // Specs can depend on the package they are testing.
+      ["@angular-devkit/build-angular"]
```

Comment:
This applies to more than just specs.  Can you add a note that this is a workaround?  I think the ideal solution would be to have a separate tslint ruleset for spec files.  There are some other rules that are commonly disabled in spec files as well (big functions, etc.).

Reply:
Replaced comment with:
```
      // Specs can depend on the package they are testing.
      // This is a workaround, we should instead have a dedicated
      // tslint config only for specs.
```